examples/regional_private_node_pool_oauth_scopes/variables.tf

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "cluster_name" {
```

Comment:
We should try to hard-code as much as possible within the example, or at least provide default values for variables. The intent of our examples is to demonstrate usage patterns rather than to be as flexible as possible. `project_id` is really the only input here which must be left as variable. Note though that hard-coded names should include a random element like a `random_pet_name` to avoid name collisions.